packages/lesswrong/server/serverStartup.ts

```diff
@@ -111,6 +107,10 @@ async function serverStartup() {
     }
   }
 
+  // eslint-disable-next-line no-console
+  console.log("Running onStartup functions");
+  await runStartupFunctions();
+
```

Comment:
Discuss this move?

Reply:
It turns out that migrating these collections requires running a startup function that relies on the internal representation of Postgres tables already being built. This just makes sure those tables are initialised before running the startup functions.